src/@utils/provider.ts

```diff
@@ -85,16 +87,44 @@ export async function getFileDidInfo(
 
 export async function getFileUrlInfo(
```

Comment:
same [as in that other PR](https://github.com/oceanprotocol/market/pull/1732#discussion_r995408419), we are extending this method to check for more than a URL, hence "Url" should be removed from the name.